### ISSUE TITLE
Give us the juicy details in the documentation

### ISSUE THREAD

**Issue by blakeperdue** [TryGhost/Ghost-Desktop#139]
I love this app. Love that it's open source. I would love it even more if you detailed some of your tech decisions making this app. How is it built? What frameworks or libraries did you use? Would love a brief overview of those kinds of details in the documentation to make it easer for folks looking at this repo to understand how they might try to pick up contributing. Thanks. :)

**Comment by felixrieseberg** [comment]
Oh man, I love that you opened this issue. As a volunteer here, that warms my heart. I'll write something up 👍

**Comment by adramalech** [comment]
So from what I can tell from just scanning through the code it is a Node.js v5.0+ Electron app using Ember.js utilizing Babel and grunt.

**Comment by blakeperdue** [comment]
Yeah, that's what I saw too. I believe Ghost is built on Ember? Seems like one solution would be to wrap the core Ghost admin UI into a webview. Does Felix's app rebuild the UI or wrap the core Ghost app?
